src/main/kotlin/com/th/plu/domain/member/MemberSocialType.kt

```diff
@@ -0,0 +1,9 @@
+package com.th.plu.domain.member
+
+import lombok.Getter
+
+@Getter
+enum class MemberSocialType(private val value: String) {
```

Comment:
```suggestion
enum class MemberSocialType(val value: String) {
```

lombok 제거, private 제거하면, 외부에서 불변한 value get 가능합니다